packages/hardhat/test/PaymentSplitterWithFeeUpgradeable.test.ts

```diff
@@ -0,0 +1,93 @@
+import chai from "chai";
+import { ethers } from "hardhat";
+import { solidity } from "ethereum-waffle";
+import { PaymentSplitterWithFeeUpgradeableDerived__factory } from "../typechain/factories/PaymentSplitterWithFeeUpgradeableDerived__factory";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { parseEther } from "ethers/lib/utils";
+import { PaymentSplitterWithFeeUpgradeableDerived } from "../typechain/PaymentSplitterWithFeeUpgradeableDerived";
+
+chai.use(solidity);
+const { expect } = chai;
+
+describe("PaymentSplitterWithFeeUpradeable", async () => {
```

Comment:
because this is an infra change, thought it could be nice to test `release` under more fee configurations like:

* zero
* 10K
* over 10K
* maybe an odd number like 333

What do you think?

Reply:
@eladmallel sounds good. I'll add these tests